index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Confidential Storage 0.1</title>
+    <title>EDV Specification 0.1</title>
```

Comment:
```suggestion
    <title>Encrypted Data Vaults v0.1</title>
```

Spell out the name of the specification.